front-end/src/App.test.tsx

```diff
@@ -1,8 +1,9 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import Header from './Header';
+import UserHomePage from './UserHomePage';
 
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  render(<Header />);
+  const linkElement = screen.getByText(/welcome back hemanth/i);
```

Comment:
Does this test even pass? ‘Welcome back Hemanth’ is rendered in the UserHomepage.tsx file not in Header

Reply:
It doesn't, Like I've mentioned before I'm still working on it.